### PR TITLE
Display mods' loading times (and fixed a typo).

### PR DESCRIPTION
**Pull request by K07H**

### CODE REVIEW

File: QModInstaller/QModPatcher.cs [hunk 1]
```diff
@@ -129,12 +140,26 @@ private static QMod LoadMod(QMod mod)
...
                     var entryType = String.Join(".", entryMethodSig.Take(entryMethodSig.Length - 1).ToArray());
                     var entryMethod = entryMethodSig[entryMethodSig.Length - 1];
 
                     MethodInfo qPatchMethod = mod.loadedAssembly.GetType(entryType).GetMethod(entryMethod);
                     qPatchMethod.Invoke(mod.loadedAssembly, new object[] { });
+
+                    // Stop Stopwatch
+                    sw.Stop();
+                    // Log elapsed time
+                    string elapsedTime = String.Format("{0:00}h{1:00}m{2:00}s{3:00}ms", sw.Elapsed.Hours, sw.Elapsed.Minutes, sw.Elapsed.Seconds, sw.Elapsed.Milliseconds / 10);
```
Comment (AlexejheroYTB-acc) [inline QModInstaller/QModPatcher.cs]: I would change this formatting

File: QModInstaller/QModPatcher.cs [hunk 2]
```diff
@@ -129,12 +140,26 @@ private static QMod LoadMod(QMod mod)
             {
                 try
                 {
+                    Console.WriteLine("QMOD INFO: Now loading mod \"" + (!String.IsNullOrEmpty(mod.DisplayName) ? mod.DisplayName : mod.AssemblyName) + "\"...");
```
Comment (AlexejheroYTB-acc) [inline QModInstaller/QModPatcher.cs]: I don't think this line is necessary

**Comment by AlexejheroYTB-acc** [comment]
Merged to the 1.3 branch

**Comment by K07H** [comment]
I strongly disagree with changes made to my PR:
We don't need 50 lines to display a simple timestamp.
It makes it really hard to read and maintain.
I'd like to have the changes reverted back, thank you :^)

**Comment by AlexejheroYTB** [comment]
This is not the final product

**Comment by AlexejheroYTB** [comment]
And it doesn't matter how many lines it takes, if it looks good.

**Comment by AlexejheroYTB** [comment]
A lot of those lines are just `{` and `}`, which could be removed

**Comment by K07H** [comment]
50 lines for printing a simple timestamp is too much lol

**Pull request merged**